packages/connect/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 
-### Added
+# Fixed
 
-- Changelog
+- Add missing subscription/unsubscription methods ([#850](https://github.com/paritytech/substrate-connect/pull/850/))
+- Wrong unsubscription JSON-RPC methods being used when calling `ProviderInterface.disconnect()` ([#842](https://github.com/paritytech/substrate-connect/pull/842)).
```

Comment:
There's no need for it to be two lines
Users don't care that we fixed a bug introduced in between two versions

```suggestion
- Fixed wrong unsubscription JSON-RPC methods being used when calling `ProviderInterface.disconnect()` ([#842](https://github.com/paritytech/substrate-connect/pull/842)).
```